events/test_routes.py

```diff
@@ -0,0 +1,85 @@
+"""Unit tests for events service HTTP routes."""
+
+import unittest
+from unittest.mock import patch
+import datetime
+from bson import json_util
+import mongomock
+from app import app
+from eventclass import Event
+
+EXAMPLE_TIME = datetime.datetime(2019, 6, 11, 10, 33, 1, 100000)
+
+VALID_INFO = {
+    'name': 'valid_event',
+    'description': 'This event is formatted correctly!',
+    'author': 'admin',
+    'event_time': EXAMPLE_TIME}
+VALID_INFO_WITH_ID = {
+    'name': 'test_event',
+    'description': 'This event is formatted correctly too!',
+    'author': 'admin',
+    'event_time': EXAMPLE_TIME,
+    'event_id': "unique_event_id"}
+INVALID_INFO_MISSING_ATTRIBUTE = {
+    'name': 'invalid_event_missing',
+    'description': 'This event is missing an author!',
+    'event_time': EXAMPLE_TIME}
+INVALID_INFO_EXTRA_ATTRIBUTE = {
+    'name': 'invalid_event_extra',
+    'description': 'This event has an extra attribute.',
+    'author': 'admin',
+    'event_time': EXAMPLE_TIME,
+    'extra_attribute': 'I am invalid'}
+
+FAKE_INFO = [
```

Comment:
Where is this used?

Reply:
I was going to use it test adding events, I'll clarify why I need both fake info and fake events

Reply:
Added comment in f73dfa0